### PR TITLE
Added launchable field and fixed whitespace/indents.

### PR DESCRIPTION
**Pull request by rrigby**
Added categories according to spec https://standards.freedesktop.org/menu-spec/latest/apas02.html

### CODE REVIEW

**Comment by rrigby** [comment]
Set diff settings to 'hide whitespace changes' to see the actual categories changes.

File: net.wz2100.wz2100.appdata.xml [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
+  <screenshots>
+    <screenshot type="default">
+      <image>http://wz2100.net/static/img/wz2100-CAM_3A50.png</image>
+    </screenshot>
+  </screenshots>
+  <project_license>GPL-2.0-or-later</project_license>
+  <developer_name>Warzone 2100 team</developer_name>
+  <url type="bugtracker">http://developer.wz2100.net/report/1</url>
+  <url type="help">http://developer.wz2100.net/wiki/NewFAQ/</url>
+  <categories>
```
Comment (TingPing) [inline net.wz2100.wz2100.appdata.xml]: Don't need categories, they are in the `.desktop` file.

File: net.wz2100.wz2100.appdata.xml [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
-	<screenshots>
-		<screenshot type="default">
-		<image>http://wz2100.net/static/img/wz2100-CAM_3A50.png</image>
-		</screenshot>
-	</screenshots>
-	<project_license>GPL-2.0-or-later</project_license>
-	<developer_name>Warzone 2100 team</developer_name>
-	<url type="bugtracker">http://developer.wz2100.net/report/1</url>
-	<url type="help">http://developer.wz2100.net/wiki/NewFAQ/</url>
+  <id>net.wz2100.wz2100</id>
```
Comment (TingPing) [inline net.wz2100.wz2100.appdata.xml:3]: This is missing `<launchable type="desktop-id">net.wz2100.wz2100.desktop</launchable>`

File: net.wz2100.wz2100.appdata.xml (hunk 1 above)
Comment (rrigby) [reply net.wz2100.wz2100.appdata.xml]: Categories match the desktop file and have different use cases for appstream than .desktop files. I think it's valid to include them.

File: net.wz2100.wz2100.appdata.xml (hunk 2 above)
Comment (rrigby) [reply net.wz2100.wz2100.appdata.xml:3]: I can add this if you wish, though I'd be adding it blind because I'm not sure what this does.

**Comment by rrigby** [comment]
@TingPing let me know if this looks correct now.

File: net.wz2100.wz2100.appdata.xml (hunk 2 above)
Comment (TingPing) [reply net.wz2100.wz2100.appdata.xml:3]: It tells it what `.desktop` file its associated with. That allows it to extract data from it and expose launching to stores.

**Pull request merged**